go.mod

```diff
@@ -4,6 +4,8 @@ go 1.17
 
 replace github.com/fluxcd/source-controller/api => ./api
 
+replace helm.sh/helm/v3 => helm.sh/helm/v3 v3.9.0-rc.1.0.20220627184942-c52cd40acef1
```

Comment:
A comment about this would be good to know when to remove this.
Link to the upstream PR for more information.

Reply:
Will do.